src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveLoad2xVectorAndUnzipTest.template

```diff
@@ -0,0 +1,566 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/******************************************************************************
+ * This file is auto-generated from a template file by the GenerateTests.csx  *
+ * script in src\tests\JIT\HardwareIntrinsics\Arm\Shared. In order to make    *
+ * changes, please update the corresponding template and run according to the *
+ * directions listed in the file.                                             *
+ ******************************************************************************/
+using System;
+using System.Numerics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using Xunit;
+
+namespace JIT.HardwareIntrinsics.Arm
+{
+    public static partial class Program
+    {
+        [Fact]
+        public static void {TestName}()
+        {
+            var test = new {TestName}Test();
+            if (test.IsSupported)
+            {
+                // Validates basic functionality works
+                test.RunBasicScenario();
+                test.RunBasicScenario_FalseMask();
+                test.RunBasicScenario_NonFaulting();
+
+                // Validates calling via reflection works
+                test.RunReflectionScenario();
+                test.RunReflectionScenario_FalseMask();
+                test.RunReflectionScenario_NonFaulting();
+
+                // Validates passing an instance member of a class works
+                test.RunClassFldScenario();
+                test.RunClassFldScenario_FalseMask();
+                test.RunClassFldScenario_NonFaulting();
+
+                // Validates passing the field of a local struct works
+                test.RunStructLclFldScenario();
+                test.RunStructLclFldScenario_FalseMask();
+                test.RunStructLclFldScenario_NonFaulting();
+
+                // Validates passing an instance member of a struct works
+                test.RunStructFldScenario();
+                test.RunStructFldScenario_FalseMask();
```

Comment:
one case for `FalseMask` and `NonFaulting` should be good enough IMO.

Reply:
@SwapnilGaikwad  - we need to add similar test cases for `Store` too.

Reply:
Should I delete the other cases then and keep RunBasicScenario_FalseMask and RunBasicScenario_NonFaulting?

Reply:
yes, that should be fine.